pyintegration/tests/test_table.py

```diff
@@ -289,15 +274,15 @@ def test_as_of_join(self):
             result_table = left_table.aj(right_table, on=["a"])
             self.assertGreater(result_table.size, 0)
             self.assertLessEqual(result_table.size, left_table.size)
-            result_table = left_table.aj(right_table, on="a")
+            result_table = left_table.aj(right_table, on="a", joins="e", match_rule=AsOfMatchRule.LESS_THAN)
```

Comment:
This test should still pass without setting this param, right? The default is LTE.

Reply:
The modified case tests both if a single string value for 'on' works and if aj can accept the non-default match rule.